### PR TITLE
fix: remove "hydra:" remaining prefix

### PR DESCRIPTION
**Pull request by vincentchalamon**
Additionally, [api-platform/admin](https://github.com/api-platform/admin) must be updated because of the following error (with `hydra_prefix: false`):

image

### CODE REVIEW

**Review by dunglas — APPROVED** [review]

File: src/State/ApiResource/Error.php [hunk 1]
```diff
@@ -98,14 +97,7 @@ public function __construct(
...
-    #[Groups(['jsonld'])]
-    public function getHydraTitle(): ?string
-    {
-        return $this->title;
-    }
-
-    #[SerializedName('hydra:description')]
+    #[SerializedName('description')]
     #[Groups(['jsonld'])]
     public function getHydraDescription(): ?string
```
Comment (soyuka) [inline src/State/ApiResource/Error.php:102]:
```suggestion
    public function getDescription(): ?string
```

+ enlever SerializedName

**Comment by burned42** [comment]
Unfortunately it seems like this introduced a BC break for me with v4.0.3 as even with `hydra_prefix: true`, the hydra prefix is gone now.

For now I brought back the old behavior by downgrading `api-platform/state` and `api-platform/validator` to v4.0.2.

**Comment by vincentchalamon** [comment]
Hi @burned42,

On which schema did you found that the hydra prefix was gone? On a resource schema (with json-ld format), the hydra prefix is still present (cf. [API Platform Demo tests](https://github.com/api-platform/demo/blob/4.0/api/tests/Api/schemas/Book/collection.json#L181-L184)). But on an error schema, the hydra prefix has been removed ([in 4.0.3](https://github.com/api-platform/core/commit/884b0331f1281b8ca5386ddf4e562ba16c3f6d59)). @soyuka can explain further more about this removal.

**Comment by burned42** [comment]
@vincentchalamon yes, that's right. Sorry if I was a bit unspecific here.

I was only talking about the error schema.

**Comment by vincentchalamon** [comment]
@burned42 @soyuka is currently working on a fix: https://github.com/api-platform/core/pull/6704

**Comment by burned42** [comment]
@vincentchalamon @soyuka awesome, thank you :)

**Comment by burned42** [comment]
As mentioned in https://github.com/api-platform/core/issues/6701#issuecomment-2407356349 it seems like the fix included in version 4.0.4 works :+1:

**Pull request merged**